parm/config/config.defaults.s2sw

```diff
@@ -15,6 +15,7 @@ min_seaice="1.0e-6"
 use_cice_alb=".true."
 
 # config.ufs  # TODO: This is hard-wired for P8 and needs to be refactored.  For now, use case C384
+# TODO: Q. for @jessicameixner-noaa: can we make these defaults in config.ufs for C384?
```

Comment:
To me, this is mostly for the commented out: 
#layout_x_gfs=24
    #layout_y_gfs=16
    #WRTTASK_PER_GROUP_GFS=86
However, with the move to 32bit this might be a little bit less.  Plus we can probably show people how to update this in config.ufs instead, so yes?

Reply:
Please follow: layout_x < layout_y. Also I am not sure how the WRTTASK_PER_GROUP_GFS=86 is coming out, it may be related to taking whole nodes, but it is really machine dependent, the number may not good for wcoss2.